rust-toolchain.toml

```diff
@@ -0,0 +1,4 @@
+[toolchain]
```

Comment:
As I remember, there was a reason not to commit toolchain file. I could be wrong.

Reply:
Actually it doesn't build without it. And I think we can use it to ensure that correct Rust version is used by everyone. If we will have any problems with this solution we can remove this file